modules/d/tools/dmd.lua

```diff
@@ -6,9 +6,10 @@
 
 	local tdmd = {}
 
-	local project = premake.project
-	local config = premake.config
-    local d = premake.modules.d
+	local p = premake
+	local project = p.project
+	local config = p.config
+    local d = p.modules.d
```

Comment:
OK... I think we should certainly make things consistent all around, according to the editorconfig file that is committed to the repository, and anyone committing to this repo should 100% use that tool in all their editors... so yeah, maybe a separate commit for that would be better.